includes/functions.php

```diff
@@ -1046,6 +1046,27 @@ function port_fill_missing(&$port, $device)
     }
 }
 
+/**
+ * returns if a port should be filtered out, like a disabled oder ignored port
+ *
+ * @param array $port
+ * @return bool
+ */
+function port_filter($port)
+{
+    // return value
+    // true:  do filter port
+    // false: do not filter port
+    $ret = true;
+
+    if ($port['ifType']) {
+        // only poll wanted interface types
+        $ret = Config::get('interface_types.' . $port['ifType']) ?: true;
+    }
+
+    return ! $ret;
```

Comment:
You could simplify this
```suggestion
    if ($port['ifType']) {
        // only poll wanted interface types
        return !Config::has('interface_types.' . $port['ifType']);
    }
    
    return false;
```

Reply:
my intention was, to keep it flexible for other filters which come later maybe.
so "result" could be overwritten

but i can change it :-)

Reply:
code changed